RELEASE.md

```diff
@@ -64,177 +72,304 @@ A release for the bpfman project is comprised of the following major components:
+described in the [Patch Branch Release](#patch-branch-release) section. Finally,
+if it is necessary to test the release automation, the simplified process
+described in the [Release Candidate Release](#release-candidate-release) section
+can be used.
 
-Additionally you can navigate to the github release page and draft a release
-generating release notes there as well.
+### Generating Release Notes
 
-### Release Steps
+The release notes are contained in `CHANGELOG` files stored in the `changelogs`
+directory of each repository.  The change log name must contain the release
+version (e.g., `CHANGELOG-v0.5.2.md`)
 
-The following steps must be done by one of the [bpfman maintainers][bpfman-team],
-please always try and cut a release candidate before pushing an official major
-or minor release.
+The format for the change log is as follows:
 
-#### RELEASE CANDIDATE Release
+- Summary of the major changes and highlights. For example: "The v0.5.2 release
+  is a patch release that introduced..."
+- Release note details.
 
-Often times cutting a release candidate is a great way to test any changes to
-our release infrastructure before cutting an official release. Make sure release
-candidate versions contain an `rc` suffix i.e `0.4.0-rc1`.  This is a lighter
-weight process meaning many of the versioned manifests do not necessarily need
-to be created.
-
-- Open an update PR that:
-    - Adds a new changelog for the release
-    - Updates the [Cargo.toml](https://github.com/bpfman/bpfman/blob/main/Cargo.toml) version for the workspace:
-        - `version = "x.x.x"`
-        - `bpfman = { version = "x.x.x", path = "./bpfman" }"`
-        - `bpfman-api = { version = "x.x.x", path = "./bpfman-api" }`
+To simplify the generation of the release notes details, we recommend using the
+GitHub release page as described below.  Note that we only use the release page
+to generate a starting point for the release notes, and don't actually create a
+tag or do a release from it. 
+
+1. Go to https://github.com/bpfman/bpfman/releases.
+2. Push the "Draft a new release" button.
+3. Enter the new release number in the "Choose a tag" pull-down.
+4. Choose the most recent release in the "Previous tag" pull-down.
+5. Push the "Generate release notes" button.
+
+The automatically generated output will likely need to be reorganized and
+cleaned up a bit, but it provides a good starting point.
+
+### Standard Release from Main Branch
+
+This section describes the standard release process used when making a release
+from the main branch and may be used for major, minor, or patch releases.  As
+mentioned above, we first complete the release for `bpfman` and then follow that
+up with a release for `bpfman-operator`.
+
+#### bpfman Release
+
+- Create a new branch in your `bpfman` fork named, for example
+  `<githubuser>/release-x.y.z`, and use the new branch in the upcoming steps.
+- Make the following changes
```

Comment:
Question:
Do you think it is worth adding a step to search though the code for the current version, `0.5.1` (without the `v`) and replace with the current release. There is documentation that references releases like this file, loading from RPM, etc, that may be nice to always point to latest release in the instructions? There are a few spots pointing to 0.4.2 currently, so if we add this step, we should do a one time scrub for spots with older releases too.

Reply:
That makes sense.

Reply:
Actually, this is a little tricky.  There are a lot of places where the old release needs to remain, and there are other dependencies that just happen to have the same version number as the current or older bpfman releases.

Reply:
So the one time scrub might a bit tricky maybe. Is there anywhere that needs to stay at 0.5.1? But then going forward, just change current release to next release everywhere

Reply:
I added a bullet, but whoever is using it will need to use their judgment.  You may find the string in the following places that shouldn't be changed: changelogs, cargo.toml, cargo.lock, Containerfile.bpfman.openshift (which has a different version), example config files (at least)